dataland-frontend/src/components/forms/parts/EuTaxonomyBasicInformation.vue

```diff
@@ -10,10 +10,10 @@
         :name="'fiscalYearDeviation'"
         :description="euTaxonomyKpiInfoMappings.fiscalYearDeviation"
         :label="euTaxonomyKpiNameMappings.fiscalYearDeviation"
-        :options="[
-          { label: 'Deviation', value: 'Deviation' },
-          { label: 'No Deviation', value: 'NoDeviation' },
-        ]"
+        :options="{
+          Deviation: 'Deviation',
+          NoDeviation: 'No Deviation',
+        }"
```

Comment:
I think this can be even more simplified by just writing:
`:options="['Deviation', 'No Deviation']`

Reply:
But then the value send to the backend is `No Deviation` instead of `NoDeviation`. We had this issue before I think.

Reply:
So, no need to change?

Reply:
Hold on I will check this.  I thought that it is already like that.

Reply:
@fsoffel  is right

The field name here (so e.g. NoDeviation) is being sent to the API, and the field value ("No Deviation") is used as the label.

So no need for a chang here.  Closing

👍